src/SprykerEco/Zed/Econda/EcondaConfig.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Zed\Econda;
+
+use Exception;
+use Spryker\Shared\Application\ApplicationConstants;
+use Spryker\Zed\Kernel\AbstractBundleConfig;
+use SprykerEco\Shared\Econda\EcondaConstants;
+
+class EcondaConfig extends AbstractBundleConfig
+{
+    public const ECONDA_CSV_DELIMITER = '|';
+    public const ECONDA_CSV_CATEGORY_DELIMITER = '^^';
+
+    /**
+     * @return string
+     */
+    public function getHostYves(): string
+    {
+        return $this->get(ApplicationConstants::HOST_YVES);
+    }
+
+    /**
+     * @return string
+     */
+    public function getFileExportPath(): string
+    {
+        return $this->get(EcondaConstants::ECONDA_CSV_FOLDER_PATH);
+    }
+
+    /**
+     * @return string
+     */
+    public function getFileExportDelimiter(): string
+    {
+        return $this->get(EcondaConstants::ECONDA_CSV_DELIMITER);
+    }
+
+    /**
+     * @param string $pdoEcondaQueryName
+     * @param string $dbEngineName
+     *
+     * @throws \Exception
+     *
+     * @return string
+     */
+    public function getPdoEcondaQueryClassName($pdoEcondaQueryName, $dbEngineName): string
```

Comment:
shouldn't this be `getEcondaPdoQueryClassName` (and everywhere else) ? Also the order of arguments is the otherway as you access the data in the array. That way feels more logical.

Reply:
done